distribution/pom.xml

```diff
@@ -34,6 +34,11 @@
             <groupId>org.freemarker</groupId>
             <artifactId>freemarker</artifactId>
         </dependency>
+        <!-- Needed here so references to MapStruct Gem classes can be resolved during JavaDoc generation -->
+        <dependency>
```

Comment:
This feels a bit weird.. Why would the javadoc processor require `gem-api`? Is there no way of telling the javadoc processor to exclude this explicitly from analysis.

Reply:
That's how it works. It looks at our sources and tries to analyze it. The same problem as above with `freemarker`.